### ISSUE TITLE
0.1.5 fails to parse JSC stack traces

### ISSUE THREAD

**Issue by ide** [errwischt/stacktrace-parser#11]
This is an example stack trace that gets parsed with 0.1.4 but not 0.1.5:
```
_exampleFunction@/home/test/project/App.js:125:13
_depRunCallbacks@/home/test/project/node_modules/dep/index.js:77:45
tryCallTwo@/home/test/project/node_modules/react-native/node_modules/promise/lib/core.js:45:5
doResolve@/home/test/project/node_modules/react-native/node_modules/promise/lib/core.js:200:13
```

In 0.1.4 you get:
```
[{
    "file": "/home/test/project/App.js",
    "methodName": "_exampleFunction",
    "lineNumber": 125,
    "column": 13
}, {
    "file": "/home/test/project/node_modules/dep/index.js",
    "methodName": "_depRunCallbacks",
    "lineNumber": 77,
    "column": 45
}, {
    "file": "/home/test/project/node_modules/react-native/node_modules/promise/lib/core.js",
    "methodName": "tryCallTwo",
    "lineNumber": 45,
    "column": 5
}, {
    "file": "/home/test/project/node_modules/react-native/node_modules/promise/lib/core.js",
    "methodName": "doResolve",
    "lineNumber": 200,
    "column": 13
}]
```

and in 0.1.5 you get an empty array.

**Comment by Calamari** [comment]
Thanks for the report, I will see what causes that regression and restore the behavior.

**Comment by Calamari** [comment]
I found the culprit and fixed it in version 0.1.6. 

I'll close this issue now. In the unlikely event that this problem persists, please say so, and I will reopen this ticket. And if you find something else, you know where to note it :)

Thanks for reporting this.
